synapse/lib/aha.py

```diff
@@ -259,6 +417,15 @@ async def _setAhaSvcDown(self, name, linkiden, network=None):
         logger.debug(f'Set [{svcfull}] offline.')
 
     async def getAhaSvc(self, name):
+
+        if name.endswith('...'):
```

Comment:
this `...` notation is used in docs and we should have a test for it.

Reply:
In the docs I definitely interpreted this as "fill in the rest of the information" rather than being literal.  Might be worth a clarifying note in the docs.